src/main/java/xyz/thepathfinder/android/ModelRegistry.java

```diff
@@ -13,6 +14,9 @@
 class ModelRegistry {
 
     private static final Logger logger = Logger.getLogger(ModelRegistry.class.getName());
+    static {
+        logger.setLevel(Level.INFO);
```

Comment:
The thing is, thats not how you're supposed to use loggers. It would be a
pain in the ass for a developer to change the logging level separate for
every logger. We don't have to worry about this now, but there are a lot of
best practices for loggers we should add eventually. Namely: 1. We should
use a facade logging framework instead of java.util.logging and 2. Logging
levels are specified in a configuration file, not hardcoded separately in
100 places in the code.

http://stackoverflow.com/questions/9740569/logger-for-java-library

On Fri, Jan 22, 2016 at 3:58 AM David Robinson <notifications@github.com>
wrote:

> In src/main/java/xyz/thepathfinder/android/ModelRegistry.java
> <https://github.com/CSSE497/pathfinder-android/pull/23#discussion_r50513867>
> :
>
> > @@ -13,6 +14,9 @@
> >  class ModelRegistry {
> >
> >      private static final Logger logger = Logger.getLogger(ModelRegistry.class.getName());
> > +    static {
> > +        logger.setLevel(Level.INFO);
>
> You can access any logger using
> Logger.getLogger(ClassName.class.getName()); it doesn't matter if it is
> private. I can still find and change it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CSSE497/pathfinder-android/pull/23/files#r50513867>.
>